examples/create-account-with-thresholdkey.js

```diff
@@ -0,0 +1,169 @@
+import {
+    Client,
+    PrivateKey,
+    AccountId,
+    Hbar,
+    AccountCreateTransaction,
+    TransferTransaction,
+    AccountBalanceQuery,
+    AccountDeleteTransaction,
+    KeyList,
+} from "@hashgraph/sdk";
+
+import dotenv from "dotenv";
+
+dotenv.config();
+
+// Removed pino logger initialization
+const logger = console;
```

Comment:
Could you please refer to the Java example to view the Hedera logger I mentioned? Additionally, please make the necessary corrections in the code.

Reply:
resolved